frontend/src/app/app.module.ts

```diff
@@ -11,35 +11,35 @@ import { CommonModule } from '@angular/common';
 import { HTTP_INTERCEPTORS, HttpClientModule } from '@angular/common/http';
 import { NgModule } from '@angular/core';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatAutocompleteModule } from '@angular/material/autocomplete';
 import { MatBadgeModule } from '@angular/material/badge';
+import { MatButtonModule } from '@angular/material/button';
 import { MatButtonToggleModule } from '@angular/material/button-toggle';
```

Comment:
URL: http://localhost:4200/settings/core/users

Issue:
- admin user is smaller than the other users
Screenshot:
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/b15fb06d-72c5-4bd2-85dd-b8c8feaa20a5)